scripts/firepit-builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10
+FROM node:12
```

Comment:
Not opposed but was there any particular reason you made this update?  Was something not working on 10?

Reply:
There's some issues with `pkg` with `node@10` and by bumping to `12` we just sidestep that ordeal.